pkg/sql/conn_executor.go

```diff
@@ -1015,6 +1018,8 @@ func (ex *connExecutor) run(ctx context.Context, cancel context.CancelFunc) erro
 
 	var draining bool
 	for {
+		// Check to see if there are any table descriptor versions to wait on
+
```

Comment:
@andreimatei I'd like to chat with you to figure out where I should put the wait for table version code.